homeassistant/components/xiaomi_miio/translations/pl.json

```diff
@@ -58,7 +58,8 @@
         "step": {
             "init": {
                 "data": {
-                    "cloud_subdevices": "U\u017cyj chmury, aby uzyska\u0107 pod\u0142\u0105czone podurz\u0105dzenia"
+                    "cloud_subdevices": "U\u017cyj chmury, aby uzyska\u0107 pod\u0142\u0105czone podurz\u0105dzenia",
+                    "lazy_discover": "U\u017cyj leniwego wykrywania, aby zoptymalizowa\u0107 nawi\u0105zywanie po\u0142\u0105czenia z urz\u0105dzeniem"
```

Comment:
Only the english version of strings should be commited in, the rest will come automatically from lokalise as they get translated.